src/sentry/static/sentry/app/views/organizationDiscover/result/utils.jsx

```diff
@@ -73,7 +73,16 @@ export function getChartDataByDay(data, query) {
       });
     }
 
-    result.push({seriesName: key, data: output[key].data});
+    result.push({seriesName: createSubstring(key), data: output[key].data});
+  }
+
+  return result;
+}
+
+function createSubstring(seriesName) {
```

Comment:
Can you also do truncation for the distribution charts? (i.e. on line 45 of this file)

Reply:
Can you rename this function to `truncateSeriesName()`? I think it's a little bit more descriptive